### PR TITLE
Added English About page.

### PR DESCRIPTION
**Pull request by tslmy**

### CODE REVIEW

File: _data/menus.yml [hunk 1]
```diff
@@ -7,6 +7,10 @@ main:
     url: "/comedians/"
     weight: 2
 
+  - name: "English"
+    url: "/english/"
```
Comment (tzhengus) [inline _data/menus.yml]: How about `/en/`?

File: english.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+intro_image_absolute: true
+intro_image_hide_on_mobile: true
+show_people: false
+---
+
+# About: Chinese Comedy at Silicon Valley
+
+Just for fun!
+
+We are a group of Chinese comedy lovers in the Silicon Valley area, mainly focusing on performing stand-up comedy shows.
```
Comment (tzhengus) [inline english.md]: We are a group of comedy lovers in the Silicon Valley area, mainly focusing on performing stand-up comedy shows in Chinese.

File: english.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+show_people: false
+---
+
+# About: Chinese Comedy at Silicon Valley
+
+Just for fun!
+
+We are a group of Chinese comedy lovers in the Silicon Valley area, mainly focusing on performing stand-up comedy shows. 
+
+For the past three years and counting, we gather together almost weekly. So far, _Chinese Comedy at Silicon Valley_ has held nearly a hundred meet-ups. More than 60 guests have performed on the stage or virtually. Videos of our previous performances have achieved 10k+ views in total.
```
Comment (tzhengus) [inline english.md]: We are almost 40k views now.

Comment (tzhengus) [reply english.md]: Well counting Bilibili we are actually really close to 50k.

**Review by edwardhgz — APPROVED** [review]
LGTM

**Pull request merged**